<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="robot_name"/>

    <group ns="/$(arg robot_name)">
        <node name="$(anon dynparam_speech_recognizer)" pkg="dynamic_reconfigure" type="dynparam"
              args="set_from_parameters speech_recognizer">
            <param name="enable" type="bool" value="true"/>
            <param name="continuous" type="bool" value="false"/>
        </node>

        <node name="$(anon dynparam_chatbot)" pkg="dynamic_reconfigure" type="dynparam"
              args="set_from_parameters chatbot">
            <param name="enable" type="bool" value="true"/>
            <param name="mute" type="bool" value="true"/>
        </node>
    </group>
</launch>